integration-tests/suites/k8s_namespace.go

```diff
@@ -104,18 +134,20 @@ func (k *K8sNamespaceTestSuite) TearDownSuite() {
 }
 
 func (k *K8sNamespaceTestSuite) TestK8sNamespace() {
+	// Sleep to ensure asynchronous container engine lookups have completed
+	time.Sleep(10 * time.Second)
```

Comment:
Why is this sleep needed now? Were you having timing issues when running locally?

Reply:
Removed this sleep. I was having issues getting the tests to pass locally, but eventually got it working.

Reply:
The sleep is still here though.

Reply:
🤦 now removed.